frontend/src/shared/components/IconBadgeList/IconBadge/IconBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import * as S from "./IconBadge.styled";
+
+interface IconBadgeProps {
+  label: string;
+  imgUrl: string;
+}
+
+function IconBadge({ label, imgUrl }: IconBadgeProps) {
+  return (
+    <S.IconBadge>
+      <img src={imgUrl} alt="label" width={24} />
```

Comment:
```suggestion
      <img src={imgUrl} alt={label} width={24} />
```

`alt` 속성이 수정되어 있지 않는 것 같네요..!😮

그리고 개인적인 취향으로는 `width` 속성을 준다면 `<S.IconBadge>`처럼 `Styled` 컴포넌트로 분리해서 사용하는게 일관성 있다고 생각합니다~🙃

Reply:
label을 문자열로 사용하고 있었네요, 일관성을 위해 Icon 컴포넌트로 분리하였습니다!